x-pack/metricbeat/module/appsearch/docker-compose.yml

```diff
@@ -2,14 +2,17 @@ version: '2.3'
 
 services:
   appsearch:
+    image: docker.elastic.co/integrations-ci/beats-appsearch:${APPSEARCH_VERSION:-7.5.0}-1
     build:
       context: ./_meta
+      args:
+        APPSEARCH_VERSION: ${APPSEARCH_VERSION:-7.5.0}
     depends_on:
       - elasticsearch
     environment:
       - "elasticsearch.host=http://elasticsearch:9200"
       - "allow_es_settings_modification=true"
-      - "JAVA_OPTS=-Xms2g -Xmx2g"
+      - "JAVA_OPTS=-Xms1g -Xmx1g"
```

Comment:
I'm not sure if this is sufficient. Look at the minimal requirements:

```
Minimum hardware:

~2GB of free RAM or 4GB if Elasticsearch is running on the same machine.
2GB for App Server.
50Mb for Filebeat and other processes.
5GB of disk space available.
```

https://swiftype.com/documentation/app-search/self-managed/installation

Reply:
1GB seems to be enough for tests, at least locally. I will increase it back if it fails with these settings.

Reply:
Reverting this change by now.